services/movie.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
```

Comment:
Optional[List[int]] is used to type hint that the parameter can be a list of integers or None. fix in all places